README.md

```diff
@@ -940,6 +940,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [mailcatcher.me](https://mailcatcher.me/) — Catches mail and serves it through a web interface.
   * [Mailcheck.ai](https://www.mailcheck.ai/) - Prevent users to sign up with temporary email addresses, 120 requests/hour (~86,400 per month)
   * [Mailchimp](https://mailchimp.com/) — 500 subscribers and 1,000 emails/month free.
+  * [Maildroppa](https://maildroppa.com) - Up to 500 subscribers and unlimited emails as well as automations for free.
```

Comment:
Look like less than 500 subs?

<img width="453" alt="Screenshot 2024-04-16 at 12 14 08" src="https://github.com/ripienaar/free-for-dev/assets/82342/906c1e8b-dbd7-43f4-950d-13261c22097d">

Reply:
True ;). Sorry I am not awake yet :)

Reply:
> True ;). Sorry I am not awake yet :)

I can tell! you fixed the wrong entry :P

Reply:
Ups. How embarassing. ADHD in action :D